src/odemis/gui/cont/tabs/sparc2_align_tab.py

```diff
@@ -1327,6 +1533,65 @@ def _onManualFocus(self, event):
             # Update GUI
             self._pfc_manual_focus = ProgressiveFutureConnector(self._mf_future, gauge)
 
+    @call_in_wx_main
+    def _onManualFocus_ext(self, event):
+        """
+        Called when manual focus btn receives an event.
+        """
+        main = self.tab_data_model.main
+        bl_ext = main.brightlight_ext
+        align_mode = self.tab_data_model.align_mode.value
+        gauge = self.panel.gauge_autofocus_ext
+        self._mf_future.cancel()  # In case it's still changing, immediately stop (the gauge)
+
+        if event.GetEventObject().GetValue():  # manual focus btn toggled
+            # Set the optical path according to the align mode
+            if align_mode == "streak-align":
+                opath = "streak-focus"
+            elif align_mode == "tunnel-lens-align":
+                opath = "spec-focus-ext"
```

Comment:
For now the  Sparc2ManualFocus (or Sparc2AutoFocus) doesn't know about "spec-focus-ext". => add it.

Reply:
I think I misunderstood 'focus_mode' and 'alignment_mode' which are essentially the same but named differently.  Removed spec-focus-ext completely and used tunnel-lens-align everywhere now.